src/sentry/sentry_apps/services/app_request/model.py

```diff
@@ -0,0 +1,34 @@
+from typing_extensions import TypedDict
+
+from sentry.hybridcloud.rpc import RpcModel
+
+
+class RpcSentryAppOrganization(RpcModel):
+    name: str
+    slug: str
+
+
+class RpcSentryAppRequestHeaders(RpcModel):
+    content_type: str
+    request_id: str
+    sentry_hook_resource: str
+    sentry_hook_signature: str
+    sentry_hook_timestamp: str
+
+
+class RpcSentryAppRequest(RpcModel):
+    date: str
+    response_code: int
+    webhook_url: str
+    organization_id: int
+    event_type: str
+    error_id: str | None = None
+    project_id: int | None = None
+    request_body: str | None = None
+    request_headers: RpcSentryAppRequestHeaders | None = None
+    response_body: str | None = None
```

Comment:
Are we displaying this at all in the updated design? If not, does it make sense to exclude this from the model?